### PR TITLE
Map Display on Index Page

### PR DESCRIPTION
**Pull request by junydania**
## What was done?
     - Added google map on the index page

## What we learnt?
    - Integrating third party api
    - Learnt how to make Capybara run javascript
    - Learnt about poltergeist

### CODE REVIEW

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -7,7 +7,11 @@
...
         - @restaurants.each do |restaurant|
           %li= restaurant[:name]
+
           %ul
             %li Address: #{restaurant[:address]}
             %li Email: #{restaurant[:email]}
             %li Phone: #{restaurant[:phone]}
+
+    %div{ :style => "width: 1000px;" }
+      #map{ :style => "width: 1024px; height: 600px;" }
```
Comment (tochman) [inline app/views/home/index.html.haml]: Please use new hash syntax

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -9,3 +9,5 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( jquery.js )
```
Comment (tochman) [inline config/initializers/assets.rb]: Why do you need to precompile jquery? Isn't it added to the assets pipeline?

File: features/geomap_index.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Feature: As a potential customer
+  As soon as I hit the application
+  I would like to see a map with my location.
+  And I would like to see restaurants in my area
+
+@javascript
+
```
Comment (tochman) [inline features/geomap_index.feature]: Remove unnecessary line breaks

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/geomap_index.feature (hunk 3 above)
Comment (junydania) [reply features/geomap_index.feature]: Done

File: config/initializers/assets.rb (hunk 2 above)
Comment (junydania) [reply config/initializers/assets.rb]: Corrected. Saw it in the assests pipeline. Thank you.

File: app/views/home/index.html.haml (hunk 1 above)
Comment (junydania) [reply app/views/home/index.html.haml]: Corrected.

**Review by tochman — DISMISSED** [review]

**Review by diraulo — APPROVED** [review]

**Pull request merged**